storagedriver/azure/blockblob.go

```diff
@@ -0,0 +1,24 @@
+package azure
+
+import (
+	"fmt"
+	"io"
+
+	azure "github.com/MSOpenTech/azure-sdk-for-go/clients/storage"
+)
+
+// azureBlockStorage is adaptor between azure.BlobStorageClient and
+// blockStorage interface.
+type azureBlockStorage struct {
+	azure.BlobStorageClient
+}
+
+func (b *azureBlockStorage) GetBlobRange(container, blob string, start, length int64) (io.ReadCloser, error) {
```

Comment:
Canonical naming for this would be a "SectionReader". The behavior is covered [here in the io package](http://golang.org/pkg/io/#SectionReader).

This comment is mostly just an FYI.